deploy/config/crds/submariner.io_clusters_crd.yaml

```diff
@@ -1,13 +1,63 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1alpha1
```

Comment:
is this v1alpha1 or v1?

Reply:
yes, the CRD has been updated to apiextensions.k8s.io/v1.
suggest to get the CRD from submariner-operator repo by using `make` command.  cc @skeeey

Reply:
ok, I will

Reply:
done